edxval/api.py

```diff
@@ -316,27 +318,42 @@ def get_url_for_profile(edx_video_id, profile):
 
 def _get_videos_for_filter(
         video_filter,
-        sort_field=None,
-        sort_dir=SortDirection.asc
+        sort_field=VideoSortField.created,
+        sort_dir=SortDirection.desc,
+        **kwargs
 ):
     """
     Returns a generator expression that contains the videos found, sorted by
     the given field and direction, with ties broken by edx_video_id to ensure a
     total order.
     """
-    videos = Video.objects.filter(**video_filter)
+    videos_qs = Video.objects.filter(**video_filter)
     if sort_field:
         # Refining by edx_video_id ensures a total order
-        videos = videos.order_by(sort_field.value, "edx_video_id")
-        if sort_dir == SortDirection.desc:
-            videos = videos.reverse()
-    return (VideoSerializer(video).data for video in videos)
+        videos_qs = videos_qs.order_by(sort_field.value, "edx_video_id")
```

Comment:
yes it expects the sort_field to be valid. no new changes have been made in this behaviour